ui/perfherder/tests/TestsView.jsx

```diff
@@ -30,10 +30,10 @@ class HeathView extends React.PureComponent {
   }
 
   componentDidMount() {
-    this.getHealthData();
+    this.getTestsData();
```

Comment:
This method name is ambiguous. Perfherder generally works with tests.
This name can falsely hint we're requesting perf signatures or bulk of perf data.
When we're actually requesting an *aggregation*/*overview* of this type of data.

Reply:
Would `getOverviewTestsData` be a more suitable name for this function?

Reply:
Yeah, sounds good. I think `getTestsOverviewData` is even better.